SYCL/ESIMD/spec_const/spec_const_long.cpp

```diff
@@ -0,0 +1,21 @@
+//==--------------- spec_const_long.cpp  - DPC++ ESIMD on-device test -----===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
+// XFAIL: windows
+// RUN: %clangxx-esimd -fsycl -I%S/.. %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// UNSUPPORTED: cuda
+
+#define DEF_VAL -99776644220011
+#define REDEF_VAL 22001144668855
+#define STORE 1
+
+typedef long spec_const_t;
```

Comment:
`long` is 32-bit on Windows. Also, let's use C++ style
```suggestion
using spec_const_t = int64_t ;
```